vendedores/src/main/kotlin/ar/edu/unahur/obj2/vendedores/Centros.kt

```diff
@@ -0,0 +1,23 @@
+package ar.edu.unahur.obj2.vendedores
+
+
+class CentroDeDistribucion ( val ciudad : Ciudad ){
+
+    val vendedores = mutableListOf<Vendedor>()
+
+    fun agregar(vendedor : Vendedor){
+        if ( vendedores.contains(vendedor)) { throw Exception("BOOM! We have problems ")   }
+        else{	vendedores.add(vendedor) }
```

Comment:
Si se ejecuta la rama verdadera del `if`, el método explota y nunca llega al `else`. Podés entonces obviarlo, te quedaría así:
```suggestion
        if ( vendedores.contains(vendedor)) { throw Exception("BOOM! We have problems ")   }
        vendedores.add(vendedor)
```